export const getUserData = async () => {
  try {
    
  } catch (error) {
    
  }
}